.travis.yml

```diff
@@ -70,18 +70,19 @@ before_install:
     - |
         if [[ $JHI_JDK = '11' ]]; then
             echo '*** Using OpenJDK 11'
-            sudo add-apt-repository ppa:openjdk-r/ppa
-            sudo apt-get update
-            sudo apt-get install -y openjdk-11-jdk
-            sudo update-java-alternatives -s java-1.11.0-openjdk-amd64
+            curl -s get.sdkman.io | bash
+            echo sdkman_auto_answer=true > ~/.sdkman/etc/config
+            source "$HOME/.sdkman/bin/sdkman-init.sh"
+            sdk install java 11.0.2-open
+            java --version
         else
             echo '*** Using OpenJDK 8 by default'
         fi
-    - sudo add-apt-repository ppa:openjdk-r/ppa -y
-    - sudo apt-get update
-    - sudo apt-get install -y openjdk-11-jdk
-    - sudo update-java-alternatives -s java-1.11.0-openjdk-amd64
-    - java -version
+    - curl -s get.sdkman.io | bash
```

Comment:
I was going to ask you why install 2 times jdk11 here, but it was already the case in current master! And it was my fault :)
Could you remove this part as it is already done above when JHI_JDK is 11

Reply:
Hello @pascalgrimaud. Sorry for the delayed response. Fixed